src/main/java/Main.java

```diff
@@ -0,0 +1,62 @@
+import feign.Feign;
+import feign.jackson.JacksonDecoder;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.openfeign.EnableFeignClients;
+import org.springframework.cloud.openfeign.support.SpringMvcContract;
+import uk.gov.hmcts.reform.fees.client.FeesApi;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+
+@SpringBootApplication
+@EnableFeignClients
+public class Main {
+    public static void main(String[] args) {
+        if (args.length < 7) {
```

Comment:
I guess this main is for testing purposes, right?

Reply:
Correctamundo